mrack.spec

```diff
@@ -0,0 +1,57 @@
+# Created by pyp2rpm-3.3.5
+%global srcname mrack
+
+Name:           %{srcname}
+Version:        0.6.0
+Release:        1%{?dist}
+Summary:        Multicloud use-case based multihost async provisioner for CIs and testing during development
+
+License:        Apache License 2.0
+URL:            https://github.com/neoave/mrack
+Source0:        https://github.com/neoave/mrack/releases/download/v%{version}/mrack-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-AsyncOpenStackClient
+BuildRequires:  beaker-client
```

Comment:
Which version of this has the f33?

Reply:
`beaker-client-28.2-1.fc33.noarch`
All versions defined in `requirements.txt` are still required and checked during package installation.